### PR TITLE
fix: remove scrollbar on x-axis in toc

### PR DESCRIPTION
**Pull request by iamsaumya**

### CODE REVIEW

**Comment by iamsaumya** [comment]
Any particular reason why we are debouncing the showToast?

**Comment by tommoor** [comment]
It was a copy/paste from `Settings -> Security`, I think it felt more natural on that screen as it has multiple checkboxes – currently a non-issue here and could be removed

File: shared/utils/parseDocumentSlug.ts [hunk 1]
```diff
@@ -11,5 +11,7 @@ export default function parseDocumentSlug(url: string) {
     }
   }
 
-  return parsed.replace(/^\/doc\//, "");
+  return parsed.lastIndexOf("/doc/") === 0
```
Comment (tommoor) [inline shared/utils/parseDocumentSlug.ts:14]: gonna need context/tests for this change 😄

Comment (iamsaumya) [reply shared/utils/parseDocumentSlug.ts:14]: `- [ ] Setup the [Slack integration](/settings/integrations/slack)`

In the Getting started doc, hovering over this led to API request with `/settings/integrations/slack` as Id.

**Pull request merged**